export/export.go

```diff
@@ -0,0 +1,61 @@
+package export
+
+import (
+	"context"
+	"fmt"
+
+	nanoid "github.com/matoous/go-nanoid"
+
+	"github.com/treeverse/lakefs/parade"
+
+	"github.com/treeverse/lakefs/catalog"
+)
+
+func getExportID(repo, branch, commitRef string) (string, error) {
+	nid, err := nanoid.Nanoid()
+	if err != nil {
+		return "", err
+	}
+	return fmt.Sprintf("%s-%s-%s-%s", repo, branch, commitRef, nid), nil
+}
+
+// ExportBranchStart inserts a start task on branch and changes branch export state to pending
+// if export on  already in progress will return error
```

Comment:
Could you rephrase this comment please?  I found it a bit hard to follow -- maybe just adding punctuation will make it easier for me!

Reply:
Done